Solinea/goldstone-server
# Saved search dropdown api/event/logs

### also has minor css change in goldstone-compliance under separate PR.

To test, pull branch and navigate to

- #reports/logbrowser
- #reports/apibrowser
- #reports/eventbrowser

and check that the predefined searches in the dropdowns are as expected.

Then...
click on the gear next to any of the predefined search dropdowns and create a new predefined search for api/log/event.
Navigate back to the reports and verify that your predefined searches show up in the dropdown list.
Also, please test updating and deletion of your own predefined searches.

**your custom predefined searches in events/api will not trigger chart changes. That functionality is ticketed and will be handled then.**

![image](https://cloud.githubusercontent.com/assets/5633015/12763306/80b6cb26-c9a8-11e5-93b7-091e881fe863.png)


![image](https://cloud.githubusercontent.com/assets/5633015/12763310/85a9cbd8-c9a8-11e5-9533-195f3b738c13.png)
